src/components/ClusterNodes/ClusterOverview.tsx

```diff
@@ -0,0 +1,227 @@
+import React, { useEffect, useState } from 'react'
+import { ERROR_TYPE } from "./types";
+import { ReactComponent as Error } from '../../assets/icons/ic-error-exclamation.svg'
+import { ReactComponent as Dropdown } from '../../assets/icons/ic-chevron-down.svg'
+import { getClusterNote } from './clusterNodes.service';
+import { useParams } from 'react-router-dom';
+import GenericDescription from '../common/Description/GenericDescription';
+import { defaultClusterNote } from './constants';
+import moment from 'moment';
+import { Moment12HourFormat } from '../../config';
+import { Progressing, showError } from '@devtron-labs/devtron-fe-common-lib';
+
+interface DescriptionDataType {
+    descriptionId: number,
+    descriptionText: string,
+    descriptionUpdatedBy: string,
+    descriptionUpdatedOn: string
+}
+
+
+export default function ClusterOverview({ isSuperAdmin, clusterCapacityData, clusterErrorList, clusterErrorTitle }) {
+    const { clusterId } = useParams<{
+        clusterId: string
+    }>()
+    const [collapsedErrorSection, setCollapsedErrorSection] = useState<boolean>(true)
+    const [descriptionData, setDiscriptionData] = useState<DescriptionDataType>({
+        descriptionId: 0,
+        descriptionText: defaultClusterNote,
+        descriptionUpdatedBy: defaultClusterNote,
+        descriptionUpdatedOn: ''
+    })
+    const [isLoading, setIsLoading] = useState(true)
+
+    useEffect(() => {
+        setIsLoading(true)
+        Promise.all([getClusterNote(clusterId)]).then(([clusterNoteData]) => {
+            if (clusterNoteData.result) {
+                const _clusterNote = clusterNoteData.result.clusterNote
+                let _moment: moment.Moment
+                let _date: string
+                const data = { ...descriptionData }
```

Comment:
Why are we spreading old data if we resetting this below

Reply:
fixed